third_party/2and3/pymysql/cursors.pyi

```diff
@@ -22,12 +22,24 @@ class Cursor:
     def executemany(self, query: str, args) -> int: ...
     def callproc(self, procname, args=...): ...
     def fetchone(self) -> Optional[Gen]: ...
-    def fetchmany(self, size: Optional[int] = ...) -> Optional[Gen]: ...
+    def fetchmany(self, size: Optional[int] = ...) -> Union[Optional[Gen], List[Gen]]: ...
     def fetchall(self) -> Optional[Tuple[Gen, ...]]: ...
     def scroll(self, value, mode=''): ...
     def __iter__(self): ...
 
 class DictCursor(Cursor):
     def fetchone(self) -> Optional[Dict[str, Any]]: ...
-    def fetchmany(self, size=None) -> Optional[Tuple[Dict[str, Any], ...]]: ...
+    def fetchmany(self, size: Optional[int] = ...) -> Optional[Tuple[Dict[str, Any], ...]]: ...
     def fetchall(self) -> Optional[Tuple[Dict[str, Any], ...]]: ...
+
+class DictCursorMixin:
+    dict_type = ...  # type: dict
+
+class SSCursor(Cursor):
+    def fetchall(self) -> Optional[Tuple[Gen, ...]]: ...
+    def fetchall_unbuffered(self) -> Iterator[Tuple[Gen, ...]]: ...
+    def __iter__(self) -> Iterator[Tuple[Gen, ...]]: ...
+    def fetchmany(self, size: Optional[int] = ...) -> List[Gen]: ...
+    def scroll(self, value, mode: str = ...): ...
```

Comment:
Missing a type for `value` and a return type.

Reply:
Pushed a commit to correct this. I copied the types from the scroll in the pre-existing class for this. I've corrected there as well.